docs/synapse/power_ups/rapid_power_ups.rst

```diff
@@ -0,0 +1,97 @@
+- `Synapse-GreyNoise <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-greynoise/index.html>`_
+- `Synapse-HybridAnalysis <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-hybridanalysis/index.html>`_
+- `Synapse-Jira <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-jira/index.html>`_
+- `Synapse-MISP (Free) <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-misp/index.html>`_
+- `Synapse-MITRE ATT&CK (Free) <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-mitre-attack/index.html>`_
+- `Synapse-MITRE-CVE <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-mitre-cve/index.html>`_
+- `Synapse-PassiveTotal <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-passivetotal/index.html>`_
+- `Synapse-Shodan <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-shodan/index.html>`_
+- `Synapse-SpyCloud <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-spycloud/index.html>`_
+- `Synapse-Tor (Free) <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-tor/index.html>`_
+- `Synapse-Twitter <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-twitter/index.html>`_
+- `Synapse-URLScan <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-urlscan/index.html>`_
+- `Synapse-Utils <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-utils/index.html>`_
+- `Synapse-Virustotal <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-virustotal/index.html>`_
+- `Synapse-VMRay <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-vmray/index.html>`_
+- `Synapse-VXIntel <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-vxintel/index.html>`_
+- `Synapse-Whoxy <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-whoxy/index.html>`_
+- `Synapse-ZETAlytics <https://commercial.docs.vertex.link/projects/rapid-powerups/en/latest/storm-packages/synapse-zetalytics/index.html>`_
+
+
+.. _rapid-powerups-getting-started:
+
+Getting Started with Rapid Power-Ups
+------------------------------------
+
+Vertex maintains a package repository which allows for loading public and private packages.
+
+If you are a :ref:`synapse-ui` user, you can navigate to the Power-Ups tab to register your Cortex and configure packages
+directly from the UI.
+
+Alternatively, one can use the `storm`_ tool can also be used.
```

Comment:
extra words in  "one can use the `storm`_ tool can also be used."

Reply:
fixed